.github/workflows/release.yml

```
@@ -112,6 +114,8 @@ jobs:
         run: |
           bun install
           bun run build
+          npm version patch
           npm publish
+          git push origin main 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)